source/backend/api/Areas/Disposition/Controllers/DispositionFileController.cs

```diff
@@ -176,6 +177,48 @@ public IActionResult GetDispositionTeamMembers()
             return new JsonResult(_mapper.Map<IEnumerable<DispositionFileTeamModel>>(team));
         }
 
+        [HttpGet("{id:long}/offers")]
+        [HasPermission(Permissions.DispositionView)]
+        [Produces("application/json")]
+        [ProducesResponseType(typeof(IEnumerable<DispositionFileOfferModel>), 200)]
+        [SwaggerOperation(Tags = new[] { "dispositionfile" })]
+        [TypeFilter(typeof(NullJsonResultFilter))]
+        public IActionResult GetDispositionFileOffers([FromRoute]long id)
+        {
+            _logger.LogInformation(
+                "Request received by Controller: {Controller}, Action: {ControllerAction}, User: {User}, DateTime: {DateTime}",
+                nameof(DispositionFileController),
+                nameof(GetDispositionFileOffers),
+                User.GetUsername(),
+                DateTime.Now);
+
+            _logger.LogInformation("Dispatching to service: {Service}", _dispositionService.GetType());
+
+            var dispositionOffers = _dispositionService.GetOffers(id);
+            return new JsonResult(_mapper.Map<IEnumerable<DispositionFileOfferModel>>(dispositionOffers));
+        }
+
+        [HttpGet("{id:long}/sales")]
+        [HasPermission(Permissions.DispositionView)]
+        [Produces("application/json")]
+        [ProducesResponseType(typeof(IEnumerable<DispositionFileSaleModel>), 200)]
+        [SwaggerOperation(Tags = new[] { "dispositionfile" })]
+        [TypeFilter(typeof(NullJsonResultFilter))]
+        public IActionResult GetDispositionFileSales([FromRoute]long id)
```

Comment:
Hmm, not sure I agree with this, if this is mapped as a 0-1 I think I would rather this return an object or null, returning an array like this isn't clear to the consumer what the data model actually is.